apollo-router/src/plugins/rhai/engine.rs

```diff
@@ -173,11 +173,22 @@ mod router_json {
     }
 }
 
+#[export_module]
+mod router_sha256 {
+    use sha2::Digest;
+
+    #[rhai_fn(pure)]
+    pub(crate) fn digest(input: &mut ImmutableString) -> String {
+        let hash = sha2::Sha256::digest(input.as_bytes());
+        hex::encode(hash)
+    }
+}
+
 #[export_module]
 mod router_expansion {
     pub(crate) type Expansion = expansion::Expansion;
 
-    #[rhai_fn(name = "get", return_raw)]
+    #[rhai_fn(name = "digest", return_raw)]
```

Comment:
Please revert this change.

Reply:
I have no idea why I had done this- probably working in the wrong spot. fixed.